lib/mirage/target/solo5.ml

```diff
@@ -152,12 +152,13 @@ let main i =
  (modes (native exe))
  (libraries %a)
  (link_flags %a -cclib "-z solo5-abi=%a")
- (modules (:standard \ config manifest))
+ (modules (:standard \ %a manifest))
  (foreign_stubs (language c) (names manifest))
 )
 |}
     (context_name i) main (pp_list "libraries") libraries (pp_list "link_flags")
-    flags Key.pp_target target
+    flags Key.pp_target target Fpath.pp
+    (Fpath.rem_ext (Fpath.base (Info.config_file i)))
```

Comment:
> Is the issue that -f should never have a directory part?

It's because the `mirage-skeleton` test gives `some/directory/config.ml` as the configuration file for unikernels but we want to refer into the generated `dune.build` to the `config.ml` located into the same directory.